internal/hud/server/websocket.go

```diff
@@ -67,6 +67,11 @@ func (ws WebsocketSubscriber) Stream(ctx context.Context, store *store.Store) {
 func (ws WebsocketSubscriber) OnChange(ctx context.Context, s store.RStore) {
 	state := s.RLockState()
 	view := webview.StateToWebView(state)
+
+	if view.NeedsAnalyticsNudge && !state.AnalyticsNudgeSurfaced {
+		// Nudge surfaced for the first time!
```

Comment:
There's sorrrt of a race here since we could call this OnChange multiple times before the AnalyticsNudgeSurfacedAction gets handled.

This isn't a bug since the handler checks `!state.AnalyticsNudgeSurfaced`, but it's probably worth tidying up the language here (not necessarily the first time) and maybe adding a comment to the handle func (to help readers understand any weirdness they might encounter), and maybe even worth storing a bool on `WebsocketSubscriber` to keep things clean and only send this action once. Or...I guess there are multiple subscribers, so that doesn't help. blurgh.